packages/cli/src/commands/election/current.ts

```diff
@@ -25,43 +34,22 @@ export default class ElectionCurrent extends BaseCommand {
   }
 
   async run() {
-    const kit = await this.getKit()
+    const client = await this.getPublicClient()
     const res = await this.parse(ElectionCurrent)
-    ux.action.start('Fetching currently elected Validators')
-    const validators = await kit.contracts.getValidators()
-    const signers = await this.getSigners()
-
+    ux.action.start('Fetching currently Elected Community Rpc Nodes')
+    // @ts-expect-error - remove when nicos branch is merged
```

Comment:
what branch?

Reply:
i thought you had a fix that made celo client able to be used where public client was the type

Reply:
I thought I did until we played around in our pairing sessions and turns out I was full of it